admin_ui_support/admin_ui_support.routing.yml

```diff
@@ -34,14 +34,14 @@ admin_ui_support.dblog_types:
     _permission: 'access site reports'
     _format: 'json'
 
-admin_ui_support.dblog.overview:
-  path: '/vfancy/admin/reports/dblog'
-  defaults:
-    _controller: '\Drupal\admin_ui_support\Controller\EmptyController::noop'
-  options:
-    _admin_related_route: 'dblog.overview'
-  requirements:
-    _permission: 'access site reports'
+#admin_ui_support.dblog.overview:
```

Comment:
We should probably just delete these for now. We can use git to go back anytime if we think it's valuable.

Reply:
Yeah let me remove this change from the PR. I needed the page to be able to see the 500s properly.